### PR TITLE
Pin: voting system

### PR DESCRIPTION
**Pull request by obiwac**
Everything should be bueno

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -5,8 +5,17 @@ const dotenv = require('dotenv')
 dotenv.config()
 
 const client = new Client({
+<<<<<<< HEAD
```
Comment (fdardenne) [inline src/bot.js:8]: You forgot this

**Review by fdardenne — CHANGES_REQUESTED** [review]

File: src/bot.js (hunk 1 above)
Comment (obiwac) [reply src/bot.js:8]: oops 2 sec

Comment (obiwac) [reply src/bot.js]: voilà

**Comment by fdardenne** [comment]
Thanks :rocket: !

**Pull request merged**